eureka/S2_calibrations/s2_calibrate.py

```diff
@@ -21,8 +21,7 @@
 from jwst.pipeline.calwebb_image2 import Image2Pipeline
 from ..lib import logedit, util
 from ..lib import manageevent as me
-from ..lib import readECF as rd
-
+from ..lib import readECF
 
 class MetaClass:
```

Comment:
No, this is what I was mentioning above that I had to leave behind for backwards compatibility otherwise `pickle` doesn't know how to open old meta save files. We can delete it in a month or so though, whenever we feel that would be safe